dot/rpc/modules/author.go

```diff
@@ -79,7 +79,7 @@ type ExtrinsicStatus struct {
 }
 
 // ExtrinsicHashResponse is used as Extrinsic hash response
-type ExtrinsicHashResponse common.Hash
+type ExtrinsicHashResponse string
```

Comment:
maybe update name and/or add comment to explain why string type and not hash type?